email_processing/ticket_description_generation/description_generator.py

```diff
@@ -10,8 +12,25 @@ class Email:
     subject: str
     body: str
 
-# TODO: create a function to prompt chatGPT given some input
+
+def get_description_from_gpt(input_data: str) -> str:
+    """Given some input_data, prompt GPT to return a description."""
+    response = openai.ChatCompletion.create(
+        model="gpt-3.5-turbo",
+        messages=[{"role": "user", "content": input_data}]
+    )
+    return response.choices[0].message.content
+
+
+def construct_basic_prompt(email: Email) -> str:
+    """Generates a basic prompt for GPT from the given email."""
+    return "Can you generate action items based on this email body:" + email.body
+
+
+def generate_description(email: Email, construct_prompt: Callable = construct_basic_prompt) -> str:
```

Comment:
Added an extra parameter `construct_prompt` so that we can experiment with various prompts at the same time without having to change the function implementation. Might not be useful in the long run after we have decided on the prompt but I definitely see a useful now

Reply:
This is a good idea, thank you.